packages/components/src/BrandMoment/BrandMoment.tsx

```diff
@@ -9,8 +9,19 @@ import { OverrideClassName } from "~types/OverrideClassName"
 import { useMediaQueries } from "~utils/useMediaQueries"
 import styles from "./BrandMoment.module.scss"
 
-export type BrandMomentProps = {
+type DeprecatedMoodProps = {
+  /** @deprecated Use `variant` instead */
   mood: "informative" | "positive" | "negative"
+  variant?: never
+}
+
+type VariantProps = {
+  variant: "informative" | "success" | "warning"
+  /** @deprecated Use `variant` instead */
```

Comment:
You don't need to define this twice

Reply:
Seems we do if we want the IDE to strike through the prop :)